src/Workspaces/Core/Portable/Remote/WellKnownServiceHubService.cs

```diff
@@ -21,7 +21,7 @@ internal enum WellKnownServiceHubService
 
         // owned by Unit Testing team:
         UnitTestingAnalysisService,
-        LiveUnitTestingService,
-        TestSourceLocationLookupService,
+        LiveUnitTestingBuildService,
```

Comment:
@tmat This service may go away at some point in 16.8. Should we worry about compat breaks when we remove the enum entry? Would it be better to move this to the end of the list for this reason?